components/dashboard/src/teams/SSO.tsx

```diff
@@ -17,11 +17,16 @@ import copy from "../images/copy.svg";
 import exclamation from "../images/exclamation.svg";
 import { OrgSettingsPage } from "./OrgSettingsPage";
 import { Heading2, Subheading } from "../components/typography/headings";
+import { EmptyMessage } from "../components/EmptyMessage";
 
 export default function SSO() {
     const currentOrg = useCurrentOrg();
 
-    return <OrgSettingsPage>{currentOrg.data && <OIDCClients organizationId={currentOrg.data?.id} />}</OrgSettingsPage>;
+    return (
+        <OrgSettingsPage title="SSO" subtitle="Configure OpenID Connect single sign-on providers.">
```

Comment:
I'd propose to drop "providers" from subtitle, and rather include the acronym used below. Maybe like this: `Configure OpenID Connect (OIDC) single sign-on.`